src/generators/AbstractGenerator.ts

```diff
@@ -97,6 +99,74 @@ export abstract class AbstractGenerator<
     return options.dependencyManager;
   }
 
+  private getConstrainedModels(inputModel: InputMetaModel): Array<{
+    constrainedModel: ConstrainedMetaModel;
+    dependencyManager: AbstractDependencyManager;
+  }> {
```

Comment:
Do you mind adding some jsdoc's for this function? Cause I have no idea why it's there for all generators and what it does 😅